src/clojure_tictactoe/tictactoe.clj

```diff
@@ -0,0 +1,20 @@
+(ns clojure-tictactoe.tictactoe
+  (:gen-class))
+
+
+(defn welcome []
+  "Welcome to the game of TicTacToe! \n")
```

Comment:
It's good to assign strings like this welcome message and the instructions below to constants. Then you can reference the constant in your tests instead of having to update the string in two separate places if you decide (or your client asks) to change up the copy later.

Reply:
^ What Hana said. Clojure actually doesn't have "constants" per se. But they do have `def`'s. Which are similar. It's like `defn` but not actually a function.